### PR TITLE
MEL-428 Adding date to metadata building.

### PR DESCRIPTION
**Pull request by jmille79**
Added license URL. Corrected minor differences to match CreativeWork format.

### CODE REVIEW

File: src/pages/__tests__/test.date.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import DateMicro from 'components/MicroComp/datemicro'
+import { shallow } from 'enzyme'
+import schema2 from 'components/Schema/basicschema2'
+import moment from 'moment'
+
+const meta = { id: 5, label: 'DateCreated', renderer: 'date', key: ['dateCreated'] }
+test('DateMicro Renders Properly', () => {
+  const wrapper = shallow(<DateMicro schema={schema2} meta={meta} />)
+  expect(wrapper.find('dt').props().className).toEqual(meta.key)
+  expect(wrapper.find('dd').text()).toEqual(moment(schema2[meta.key]).format('YYYY-MM-DD'))
```
Comment (danthewolfe) [inline src/pages/__tests__/test.date.js]: Let's not import `schema2` and `moment` here but instead hard code in some test schema values into the test itself and test the `toEqual` to the literal expected output.

File: src/pages/__tests__/test.default.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import Default from 'components/MicroComp/default'
 import { shallow } from 'enzyme'
 import schema2 from 'components/Schema/basicschema2'
```
Comment (danthewolfe) [inline src/pages/__tests__/test.default.js]: Same idea here. Data for the tests should be defined within the test as a `const`. I would envision the basicschema files being deleted after we move these components into production.

File: src/pages/__tests__/test.title.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import TitleMicro from 'components/MicroComp/titlemicro'
 import { shallow } from 'enzyme'
 import schema2 from 'components/Schema/basicschema2'
```
Comment (danthewolfe) [inline src/pages/__tests__/test.title.js]: Same here. Another note, we will want to also make sure each test has a test for when it should render null.

File: src/components/MicroComp/linkmicro.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const LinkMicro = ({ meta, schema }) => {
+  return (
+    <React.Fragment>
```
Comment (danthewolfe) [inline src/components/MicroComp/linkmicro.js]: We don't need the `<React.Fragment />` here since there is only one child.
